app/src/test/java/io/github/jbellis/brokk/analyzer/TreeSitterAnalyzerRustTest.java

```diff
@@ -155,24 +155,51 @@ pub enum Color {
     void testGetSkeletonHeader_Rust() {
         Optional<String> pointHeader = rsAnalyzer.getSkeletonHeader("Point");
         assertTrue(pointHeader.isPresent(), "Skeleton header for Point should be found.");
-        // The header will be the first line of the combined skeleton, which starts with the struct definition
-        assertEquals("pub struct Point {", pointHeader.get().trim());
+        assertEquals("""
```

Comment:
@cmanto I think this might be an example of what you mean by children of a class are not being correctly determined for some languages? Here `Point` and friends are siblings.